script.js

```diff
@@ -1,12 +1,138 @@
-//You can edit ALL of the code here
+let container = document.getElementById("root");
+let searchContainer = document.getElementById("search-container");
+let allEpisodes;
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  makeDropdownMenu(allEpisodes);
+  searchForEpisode();
 }
 
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  let rowMainElem = document.createElement("section");
+  rowMainElem.classList.add("row");
+
+  for (let episode of episodeList) {
+    let episodeContainer = document.createElement("div");
+    episodeContainer.classList.add("col-4");
+
+    let title = document.createElement("h4");
+    let seasonEpisodeNumber = document.createElement("h5");
+
+    let imageDiv = document.createElement("div");
```

Comment:
I'm being a bit picky here since the rest of your variable naming is very good.

Try not to use the syntax of code in your variable names (`imageDiv`). Because if you change the element type later, you'll have to change the variable name too.

Something like `imageContainer` would be better.